src/css/components/icon.scss

```diff
@@ -24,16 +24,27 @@
 .icon {
   display: inline-block;
   fill: transparent;
-  height: 1em;
-  max-height: 2rem;
-  max-width: 2rem;
-  min-height: 20px;
-  min-width: 20px;
-  stroke: currentColor;
   position: relative;
+  stroke: currentColor;
   top: -0.0625rem;
   vertical-align: middle;
   width: 1em;
+
+  &-small {
+    @extend .icon;
+    height: $grid-2;
+    width: $grid-2;
+  }
+  &-medium {
+    @extend .icon;
+    height: $grid-3;
+    width: $grid-3;
+  }
+  &-large {
+    @extend .icon;
```

Comment:
Yeah, I've noted that and understand the impulse — extends can tend to mess with the cascade and don't work across a media query boundary. However, within limits, they seem pretty reasonable — for neighboring elements in the sheet, those that don't involve media queries, and those that have no rule overlap. Should they be used here? IDK — I could always rewrite them with a copy/paste, or assume the designer would use both classes.